bundles/io.openliberty.tools.eclipse.ui/src/io/openliberty/tools/eclipse/DevModeOperations.java

```diff
@@ -909,4 +1098,11 @@ public void refreshDashboardView(boolean reportError) {
             dashboardView.refreshDashboardView(projectModel, reportError);
         }
     }
+
+    /**
+     * Cancel running jobs and avoid error message, e.g. on closing Eclipse IDE
+     */
+    public void cancelRunningJobs() {
```

Comment:
This method seems like it should be removing the jobs from the map and cancelling them.

Reply:
OK, thought this map was going away anyway but I'll add a clear() at the end.

Reply:
Actually, the JobChangeAdapter.done() will remove the job so it's not really needed.

Reply:
Ok, that sounds good.